apps/mitt-konto/src/MittKonto/Wrappers/RouteWrapper.purs

```diff
@@ -0,0 +1,126 @@
+module MittKonto.Wrappers.RouteWrapper where
+
+import Prelude
+
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import KSF.Grid as Grid
+import MittKonto.Wrappers.Elements (AutoClose(..), CloseType(..))
+import React.Basic.Classic (JSX, make)
+import React.Basic.Classic as React
+import React.Basic.DOM as DOM
+import React.Basic.Events (handler_)
+import React.Basic.Router as Router
+
+type Props p =
+  { content :: p
+  , closeType :: CloseType
+  , route :: String
+  , routeFrom :: String
+  }
+
+type State =
+  { closeable :: Boolean
+  , closeAutomatically :: AutoClose
+  , titleText :: String
+  , renderedContent :: JSX
+  , onClose :: Effect Unit
+  }
+
+type SetRouteWrapperState = (State -> State) -> Effect Unit
+
+class RouteWrapperContent p where
+  instantiate :: p -> SetRouteWrapperState -> Effect Unit
+
+component :: forall p. React.Component (Props p)
+component = React.createComponent "RouteWrapper"
+
+routeWrapper :: forall p. (RouteWrapperContent p) => (Props p) -> JSX
+routeWrapper = make component
+  { initialState
+  , didMount
+  , render
+  }
+  where
+    initialState :: State
+    initialState =
+      { closeable: true
+      , closeAutomatically: Off
+      , titleText: mempty
+      , renderedContent: mempty
+      , onClose: pure unit
+      }
+
+    didMount :: React.Self (Props p) State -> Effect Unit
+    didMount self@{ props: { content }, setState } = do
+      instantiate content setState
+
+    render :: React.Self (Props p) State -> JSX
+    render self@{ props: { closeType, route, routeFrom }, state: { closeable, closeAutomatically, titleText, onClose, renderedContent } } =
+      Router.route
+        { exact: true
+        , path: Just route
+        , render: \_ -> DOM.div
+            { className: "route-wrapper"
+            , children:
+                [ header
+                , renderedContent
+                ] <> [ autoClose self.props closeAutomatically ]
+            }
+        }
+      where
+        header :: JSX
+        header = Grid.row_
```

Comment:
I would not like to use the `Grid` module we have anymore in new code as it's dependent on [basscss](https://basscss.com/) which we don't need IMO. I think it's better to use plain css grids as they are really straight forward.